src/msg/async/dpdk/DPDK.cc

```diff
@@ -529,31 +532,21 @@ bool DPDKQueuePair::init_rx_mbuf_pool()
     // 2) Bind data buffers to each of them.
     // 3) Return them back to the pool.
     //
-    for (int i = 0; i < mbufs_per_queue_rx; i++) {
-      rte_mbuf* m = rte_pktmbuf_alloc(_pktmbuf_pool_rx);
-      assert(m);
-      rte_mbuf_refcnt_set(m, 1);
-      _rx_free_bufs.push_back(m);
+    int ret = snprintf(mz_name, sizeof(mz_name),
+         "%s",  "rx_buffer_data" + std::to_string(_qid));
```

Comment:
@liu-chunmei i don't think this compiles. 
```"rx_buffer_data" + std::to_string(_qid))``` evaluates to a `std::string`,

you might want to put something like:
```c++
string mz_name = "rx_buffer_data" + std::to_string(_qid));
// ...
const struct rte_memzone *mz = rte_memzone_reserve_aligned(mz_name.c_str(),
    mbuf_data_size*bufs_count,
    _pktmbuf_pool_rx->socket_id, mz_flags, mbuf_data_size);
```

Reply:
ah....  yes. @liu-chunmei